test/jasmine/tests/legend_test.js

```diff
@@ -524,18 +524,36 @@ describe('legend relayout update', function() {
     afterEach(destroyGraphDiv);
 
     it('should hide and show the legend', function(done) {
-        var mockCopy = Lib.extendDeep({}, mock);
```

Comment:
Is this commit locking down fixes from this PR, or just improving our test case coverage?

Reply:
Pretty sure at least one of these was broken before this PR, but I don't recall which anymore.